vic/drivers/image/src/vic_init_output.c

```diff
@@ -193,10 +207,167 @@ initialize_history_file(nc_file_struct *nc)
         log_err("Error defining time dimenension in %s", nc->fname);
     }
 
+    // define the netcdf variable time
+    status = nc_def_var(nc->nc_id, "time", NC_INT, 1,
+                        &(nc->time_dimid), &(time_var_id));
+    if (status != NC_NOERR) {
+        log_err("Error defining time variable in %s", nc->fname);
+    }
+    status = nc_put_att_text(nc->nc_id, time_var_id, "standard_name",
+                             strlen("time"), "time");
+    if (status != NC_NOERR) {
+        log_err("Error adding attribute in %s", nc->fname);
+    }
+    if (global_param.output_steps_per_day == 1) {
+        sprintf(strUnit, "days");
+    }
+    else {
+        sprintf(strUnit, "hours");
```

Comment:
Ah thats what you mean, I understood it different. Thats indeed better I will change it accordingly.